src/main/java/ru/yandex/practicum/filmorate/service/FilmService.java

```diff
@@ -0,0 +1,61 @@
+package ru.yandex.practicum.filmorate.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.yandex.practicum.filmorate.exception.NotFoundException;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.storage.impl.InMemoryFilmStorage;
+import ru.yandex.practicum.filmorate.storage.impl.InMemoryUserStorage;
+
+import java.util.Collection;
+import java.util.Comparator;
+
+@Service
+@RequiredArgsConstructor
+public class FilmService {
+    private final InMemoryFilmStorage inMemoryFilmStorage;
```

Comment:
```suggestion
    private final FilmStorage inMemoryFilmStorage;
```
Делай внедрение зависимостей через типы, так рекомендуется документацией